sigs.yaml

```diff
@@ -2157,6 +2157,13 @@ sigs:
       slack: sig-security-docs
     owners:
     - https://raw.githubusercontent.com/kubernetes/community/master/sig-security/sig-security-docs/OWNERS
+  - name: security-tooling
+    description: Development and Enhancements of Security Tooling
+    contact:
+      slack: sig-security-tooling
+      mailing_list: security-tooling-private@kubernetes.io
```

Comment:
ah, I think I found it at https://github.com/kubernetes/community/pull/5853 ... this is the list that automated vulnerability scans report to?

if it's listed here as a contact address, I suspect humans will try to join it and use it to reach out to this subproject... is that the intended use?

Reply:
Good catch on this one, Jordan.

I think we should just remove the mailing_list entry, because we use slack as a primary async communication medium, and for those things that really require a list, we currently all share the main SIG list.

Reply:
that sounds reasonable

Reply:
Yes agree. Removing the mailing list makes sense.

Reply:
Removed it now.